src/sidebar/components/Annotation/AnnotationShareControl.tsx

```diff
@@ -1,37 +1,45 @@
+import { useElementShouldClose } from '@hypothesis/frontend-shared';
 import {
   Card,
   IconButton,
-  TextInput,
-  TextInputWithButton,
-  useElementShouldClose,
-} from '@hypothesis/frontend-shared';
+  Input,
+  InputGroup,
+  CopyIcon,
```

Comment:
NB: The new batch of shared components provides one component per icon, whereas previously we had a single `Icon` component and it took a `name` prop